### PR TITLE
Updated display message of cargo metadata --help

### PR DESCRIPTION
**Pull request by bishtpawan**
Resolved Issue #8928

### CODE REVIEW

File: crates/mdman/doc/out/mdman.md [hunk 1]
```diff
@@ -54,8 +54,8 @@ man page:
 
 <dl>
 
-<dt><code>-t</code> <em>type</em></dt>
-<dd>Specifies the output type. The following output types are supported:</p>
+<dt class="option-term" id="option-mdman--t"><a class="option-anchor" href="#option-mdman--t"></a><code>-t</code> <em>type</em></dt>
```
Comment (ehuss) [inline crates/mdman/doc/out/mdman.md:57]: These seem to be some unrelated changes?

**Review by ehuss — COMMENTED** [review]
Thanks!  It looks like you were maybe trying to update the man pages?  There is some documentation [here](https://github.com/rust-lang/cargo/tree/master/src/doc) on how they work.  The file that needs to be updated is https://github.com/rust-lang/cargo/blob/master/src/doc/man/includes/options-display.md, and then you can run the `build-man.sh` script.

File: crates/mdman/doc/out/mdman.md (hunk 1 above)
Comment (bishtpawan) [reply crates/mdman/doc/out/mdman.md:57]: Yes, I thought we need to update the man page if we are updating any details. That's why this file got updated.
No worries I'll remove these changes. 

Thanks for your response.

**Comment by bishtpawan** [comment]
Closing this PR. As I've raised a new one with the only required change. https://github.com/rust-lang/cargo/pull/9050
